q2_diversity/_core_metrics.py

```diff
@@ -43,24 +46,24 @@ def core_metrics(ctx, table, sampling_depth, metadata, with_replacement=False,
 
 def core_metrics_phylogenetic(ctx, table, phylogeny, sampling_depth, metadata,
                               n_jobs=1):
-    alpha_phylogenetic = ctx.get_action('diversity', 'alpha_phylogenetic')
-    beta_phylogenetic = ctx.get_action('diversity', 'beta_phylogenetic')
+    faith_pd = ctx.get_action('diversity_lib', 'faith_pd')
+    unweighted_unifrac = ctx.get_action('diversity_lib', 'unweighted_unifrac')
+    weighted_unifrac = ctx.get_action('diversity_lib', 'weighted_unifrac')
     pcoa = ctx.get_action('diversity', 'pcoa')
     emperor_plot = ctx.get_action('emperor', 'plot')
     core_metrics = ctx.get_action('diversity', 'core_metrics')
 
     cr = core_metrics(table=table, sampling_depth=sampling_depth,
                       metadata=metadata, n_jobs=n_jobs)
 
-    faith_pd_vector, = alpha_phylogenetic(table=cr.rarefied_table,
-                                          phylogeny=phylogeny,
-                                          metric='faith_pd')
+    faith_pd_vector, = faith_pd(table=cr.rarefied_table,
+                                phylogeny=phylogeny)
 
     dms = []
-    dms += beta_phylogenetic(table=cr.rarefied_table, phylogeny=phylogeny,
-                             metric='unweighted_unifrac', n_jobs=n_jobs)
-    dms += beta_phylogenetic(table=cr.rarefied_table, phylogeny=phylogeny,
-                             metric='weighted_unifrac')
+    dms += unweighted_unifrac(table=cr.rarefied_table, phylogeny=phylogeny,
+                              n_jobs=n_jobs)
+    dms += weighted_unifrac(table=cr.rarefied_table, phylogeny=phylogeny,
+                            n_jobs=n_jobs)
```

Comment:
These need to be using the param named `threads`, not `n_jobs`, in order to match what you have done in https://github.com/qiime2/q2-diversity-lib/pull/6